back/engines/commercial/idea_assignment/app/models/idea_assignment/email_campaigns/campaigns/idea_assigned_to_you.rb

```diff
@@ -56,19 +56,19 @@ def self.category
       end
 
       def self.recipient_role_multiloc_key
-        'email_campaigns.admin_labels.recipient_role.admins_and_moderators'
+        'email_campaigns.admin_labels.recipient_role.admins_and_managers'
       end
 
       def self.recipient_segment_multiloc_key
-        'email_campaigns.admin_labels.recipient_segment.user_assigned_to_the_idea'
+        'email_campaigns.admin_labels.recipient_segment.admins_and_managers_assigned_to_a_proposal'
```

Comment:
why `...assigned to a proposal` and not `...to_an _idea`?

Reply:
nice catch!